engine/src/main/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProvider.java

```diff
@@ -160,7 +161,18 @@ public void initCounters() {
     @CacheResult(cacheName = "rbac-recipient-users-provider-get-group-users")
     public List<User> getGroupUsers(String accountId, boolean adminOnly, UUID groupId) {
         Timer.Sample getGroupUsersTotalTimer = Timer.start(meterRegistry);
-        RbacGroup rbacGroup = retryOnRbacError(() -> rbacServiceToService.getGroup(accountId, groupId));
+        RbacGroup rbacGroup;
+        try {
+            rbacGroup = retryOnRbacError(() -> rbacServiceToService.getGroup(accountId, groupId));
+        } catch (ClientWebApplicationException exception) {
```

Comment:
I searched for this exception in the jira, kibana (prod) and code, but did not find any occurrence. I guess the Quarkus rest client wraps every exception/failure as a ClientWebApplicationException, but did not find hints in the docs for it.

Reply:
I don't think we had this exception yet, but I'm preparing because the client can indeed return that (as show in the tests) when we get a 404.

Reply:
Ok.